### PR TITLE
[4.0.0] Displaying comments with User Information in REST APIs

### PR DESCRIPTION
**Pull request by Sarangan0219**
## Purpose

Related to https://github.com/wso2/product-apim/issues/9595
Doc Changes for https://github.com/wso2/carbon-apimgt/pull/9454

## Affected Versions

APIM 4.0.0

### CODE REVIEW

File: en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml [hunk 1]
```diff
@@ -4572,6 +4576,18 @@ components:
             example: admin
           apiUUID:
             type: string
+    CommenterInfo:
+      type: object
+      properties:
+        firstName:
+          type: string
+          example: John
+        LastName:
```
Comment (malinthaprasan) [inline en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml:4585]: lastName

File: en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml [hunk 2]
```diff
@@ -5236,6 +5252,14 @@ components:
       required: true
       schema:
         type: string
+    includeCommenterInfo:
+      name: includeCommenterInfo
+      in: query
+      description: |
+        Whether we need to display commentor details.
```
Comment (malinthaprasan) [inline en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml:5259]:
```suggestion
        Specifies whether to include commentor details.
```
Or use a better discription

File: en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml [hunk 3]
```diff
@@ -851,7 +852,7 @@ paths:
       - OAuth2Security: []
       x-code-samples:
       - lang: Curl
-        source: curl -k "https://localhost:9443/api/am/store/v1/apis/e93fb282-b456-48fc-8981-003fb89086ae/comments"
+        source: curl -k "https://localhost:9443/api/am/store/v1/apis/e93fb282-b456-48fc-8981-003fb89086ae/comments?includeCommenterInfo=true"
```
Comment (malinthaprasan) [inline en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml:855]:
```suggestion
        source: curl -k "https://localhost:9443/api/am/store/v1/apis/e93fb282-b456-48fc-8981-003fb89086ae/comments"
```

File: en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml (hunk 1 above)
Comment (Sarangan0219) [reply en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml]: https://github.com/wso2/docs-apim/pull/2477/commits/d6a36d4b997ad585b9c66431a8e4e748a0314836

File: en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml (hunk 2 above)
Comment (Sarangan0219) [reply en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml]: https://github.com/wso2/docs-apim/pull/2477/commits/d6a36d4b997ad585b9c66431a8e4e748a0314836

File: en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml (hunk 3 above)
Comment (Sarangan0219) [reply en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml]: https://github.com/wso2/docs-apim/pull/2477/commits/1f2967ae58194a7f67544c5fc463ed2589f2af3e

**Review by malinthaprasan — APPROVED** [review]

**Pull request merged**